projects/core/src/configurator/commons/facade/configurator-group-status.service.ts

```diff
@@ -78,9 +78,12 @@ export class ConfiguratorGroupStatusService {
   checkIsGroupComplete(group: Configurator.Group): Boolean {
     let isGroupComplete = true;
```

Comment:
I would rather put it like this:
    return group.attributes
      ? group.attributes.filter(
          (attribute) =>
            attribute.hasConflicts ||
            (attribute.required && attribute.incomplete)
        ).length === 0
      : true;

which seems easier to understand, do you agree?

Reply:
Yes I aggree, I changed that part